### ISSUE TITLE
Syntax error occurs when running cosmos

### ISSUE THREAD

**Issue by dfedynich** [react-cosmos/react-cosmos#434]
### What's up?

After running cosmos command from CLI I get the next error 

```
...\node_modules\react-cosmos-webpack\bin\cosmos.js:3
const {
      ^

SyntaxError: Unexpected token {
```

### Mkay, tell me more...

- there are two related items in my package.json:
    "react-cosmos-redux-proxy": "2.0.0-beta.24",
    "react-cosmos-webpack": "2.0.0-beta.25"
- node version 5.12.0
- npm version 3.8.6

**Comment by skidding** [comment]
Oh, I forgot the files from /bin aren't compiled. My bad completely, I made this mistake recently.

Travis only tests on Node 6, 7 and 8, so this is why this went through.

If you want to make a fast contribution, you can replace `const` with `var` in [bin/cosmos](https://github.com/react-cosmos/react-cosmos/blob/master/packages/react-cosmos-webpack/bin/cosmos.js) and [cosmos-export](https://github.com/react-cosmos/react-cosmos/blob/master/packages/react-cosmos-webpack/bin/cosmos-export.js).

Stuff like this
```
const {
  default: moduleExists
} = require('react-cosmos-utils/lib/module-exists');
```
Becomes
```
var moduleExists = require('react-cosmos-utils/lib/module-exists').default;
```

**Comment by skidding** [comment]
Actually `const` should work in Node 5 (even 4) as well. It's the destructuring that breaks. So just [this](https://github.com/react-cosmos/react-cosmos/blob/9ef3dfe013e6ffadf83cba957561604b63418c92/packages/react-cosmos-webpack/bin/cosmos.js#L3-L8) and [this](https://github.com/react-cosmos/react-cosmos/blob/9ef3dfe013e6ffadf83cba957561604b63418c92/packages/react-cosmos-webpack/bin/cosmos-export.js#L3-L8) needs to be replaced.

**Comment by skidding** [comment]
Fixed by #436
